.github/workflows/release.yml

```diff
@@ -210,25 +210,24 @@ jobs:
           if [ "$TICKET_TYPE" == "Epic" ]; then
```

Comment:
1. The URL in the curl command has been modified to remove the 'self' field from the fields parameter in the JQL query.
2. The jq command used to extract data from the Jira API response has been modified to include only the issue key, status, and assignee name.
3. The AWK script used to format the extracted data into a table has been modified to handle the changes in the extracted data structure. The issue_key variable is introduced to capture the key of the issue, and assignee is formatted with an '@' symbol before the assignee name instead of including the assignee URL. The table format is adjusted accordingly.
4. The changes seem to simplify the data extraction process and improve the readability of the table format by removing unnecessary information.